package.json

```diff
@@ -59,7 +58,9 @@
     "parserOptions": {
       "ecmaVersion": 2020
     },
-    "rules": {},
+    "rules": {
+      "no-lonely-if": 0
```

Comment:
Let’s not disable this. I’d recommend reading up on why this rule exist in the first place (if you haven’t done that already) 😊